test/emu_sv/test_state.py

```diff
@@ -9,6 +9,9 @@
 device = "cpu"
 # device= "cuda"
 
+device = "cpu"  # "cuda" if torch.cuda.is_available() else "cpu"
```

Comment:
what is the meaning of this? should the commented code replace "cpu"?

Reply:
A way to work around that, is to always generate random tensors on cpu, and move them to `device` after. It would make the tests a bit better, since running them locally would test gpu behaviour,  but we can do it later.